client/state/selectors/get-allow-restore.js

```diff
@@ -1,35 +0,0 @@
-import { createSelector } from '@automattic/state-utils';
```

Comment:
Last reference to get-allow-restore removed in #54973

Reply:
Thanks! I was not able to track that down.